spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -345,6 +345,7 @@
 "Open Software License 3.0 (OSL-3.0)": OSL-3.0
 "Open Software License v. 3.0": OSL-3.0
 "Oracle Free Use Terms and Conditions (FUTC)": LicenseRef-ort-oracle-futc
+"OSI Approved": NONE
```

Comment:
`NONE` actually it to be used "if the package contains no license information whatsoever", but "OSI Approved" actually means there *is* license information available, but we simply do not know *which* (OSI approved) license in particular.

So bottom line, I don't think we can have a global mapping for this, but need a package-specific curation that defined what exactly *is* the OSI approved license being used in this package.

Reply:
My rational for suggesting this entry was that "OSI Approved" does not denote any license. So it's invalid.
I believe this entry makes sense.

Reply:
But we're actually loosing information this way, and it's also not logically correct IMO, as there is a difference between "there is a license, it's OSI approved, but we don't tell you exactly which one it is" compared to "no license information at all is available".

Reply:
The author of the software appears to think that "OSI Approved" is a licence or that the user should choose. Frank's suggestion is good, IMHO, because "OSI Approved" is not a licence under any circumstances, so ORT shouldn't report it as such. If it's not reported as a 'detected licence', we won't waste time on it and can concentrate on any others that ORT finds.

Reply:
> ...a package-specific curation that defined what exactly is the OSI approved license being used in this package.

We usually don't do curations for declared licenses as they should reflect what is actually defined in the metadata of the package, if the package has any other license information it should be detected by the scanner.

Reply:
> "no license information at all is available".

This entry does not say that "no license information at all is available". It just says that "OSI approved" does not provide any license information. If there would be further licenses declared these would still be considered.

Reply:
I'm still not 100% convinced as the SPDX specs literally says `NONE` means that "the *package* contains no license information *whatsoever*", which is not true here. The specs do *not* say to use `NONE` if "the license field does not contain any valuable or specific information".

Anyway, *if* we do this, I believe this requires to provide a rationale in the commit message that explains why "NONE" is better than "OSI Approved" in our context, although "NONE" is less specific.

Reply:
The entries in this files map a name to a licence name or none, in the latter case indicating that the name does not represent a licence. Even if the SPDX specs talk about packages, the items in this file are not about packages, but specific licences. 

I agree that the comment can be improved, but I suggest it should say that "OSI Approved" is not a licence name and that is what NONE denotes.